packages/@aws-cdk/aws-ecs-patterns/lib/base/queue-processing-service-base.ts

```diff
@@ -45,6 +45,13 @@ export interface QueueProcessingServiceBaseProps {
    */
   readonly environment?: { [key: string]: string };
 
+  /**
+   * Secret environment variables to pass to the container
+   *
+   * @default - No secret environment variables.
+   */
+  readonly secrets?: { [key: string]: ecs.Secret };
```

Comment:
Feels weird that these are replicated across all Props... Can we create a base interface?

Reply:
OK, will move `environment` and other repetitions there too.